### ISSUE TITLE
cypress resolves remoteHost "localhost:3000" to 127.0.0.1:3000 

### ISSUE THREAD

**Issue by harz87** [cypress-io/cypress#112]
I have an rails app with puma 3.0.2, ruby 2.2.4p230 and Rails 4.2.5.2. And the server running
If I try to run this test within cypress.

```coffeescript
context "dashboard", ->
  beforeEach ->
    cy.visit("http://localhost:3000/#/smaply/projects")

  it "is dashboard", ->
    expect(true).to.be.true
```

It still doesn't go to localhost:3000 as it resolves the url 127.0.0.1:3000 which doesn't work either in the browser. This might be related to this issue in [rails](https://github.com/rails/rails/issues/19815)

**Comment by brian-mann** [comment]
This is an issue with the `puma` webserver not binding to `127.0.0.1` also noted here: https://github.com/ember-cli/ember-cli/issues/2871

Apparently when `node` makes the dns lookup for `localhost` it resolves to `127.0.0.1` which is why it makes the request there. I'm looking into seeing if we can alter that behavior.

**Comment by brian-mann** [comment]
This will be fixed in `0.14.1`.

**Comment by brian-mann** [comment]
Fixed in [`0.14.1`](https://github.com/cypress-io/cypress/wiki/changelog#0141-03132016).
